deeprank/learn/DataSet.py

```diff
@@ -1179,52 +1032,6 @@ def load_one_molecule(self, fname, mol=None):
         return (np.array(feature).astype(outtype),
                 np.array([target]).astype(outtype))
```

Comment:
1. Line 949: we would have one target per variant, instead of molecule
2. Line 768: There would be only two values 0.0 and 1.0 so min/max for targets may not be required
3. Line 833-885: We don't have a list of targets. So target ordering, ranking, normalization may not be required

Reply:
I've changed the name of molecule to variant. (1) About (2) and (3) I'm not sure. Can those unnecessary steps do harm?

Reply:
They might throw errors I suppose, maybe semantic?

Reply:
Or we could set normalize_targets to False, target_ordering to None

Reply:
We can set normalize_targets= False and target_ordering=None
This should be fine I suppose. 
I'll change the associated readme file for this.